### PR TITLE
improving the documentation for Buienradar condition codes

### PR DESCRIPTION
**Pull request by dennis-bell**
## Proposed change
Improving the documentation of the Buienradar condition codes

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @dennis-bell, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23040).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/buienradar.markdown [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following entities will be created:
...
 - **Barometer forecast**: A numeric barometric forecast (1 to 7)
 - **Barometer forecast name**: A textual representation of the barometer forecast (eg: Thunderstorms, Stable, etc.)
-- **Condition code**: A symbol and a unique code identifying the current weather condition (`[a..z]`)
+- **Condition code**: A symbol and a unique code identifying the current weather condition (`a`: (sunny/clear),`b`: (Mix of clear and medium or low clouds), `j`: (Mix of clear and high clouds),`o/r`: (Partly cloudy) increasing character increases the condition, `c`: (Heavily clouded), `p`: (Cloudy), `d`: (Alternating cloudy with local fog), `n`: (Clear and local mist or fog), `f`: (Alternatingly cloudy with some light rain), `h/k/l`: (rainy) increasing character increases the condition, `q`: Heavily clouded with rain, `w`: (Heavily clouded with rain and winter precipitation), `m`: (Heavily clouded with some light rain), `u`: (Cloudy with light snow), `i/v`: (Heavily clouded with light snowfall) increasing character increases the condition, `t`: (Heavy snowfall), `g`: (Clear with (possibly) some heavy lightning), `s`: (Cloudy with (possibly) some heavy (thunderstorms) showers)
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: I don't think this is particularly readable:

image

As a suggestion, maybe make it an indented list of the condition code item?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/buienradar.markdown (hunk 1 above)
Comment (dennis-bell) [reply source/_integrations/buienradar.markdown]: Good point!
Implemented your requested changes..
;-) first HA contribution

File: source/_integrations/buienradar.markdown [hunk 2]
```diff
@@ -48,7 +48,25 @@ The following entities will be created:
...
+  * `f`: (Alternatingly cloudy with some light rain)
+  * `h/k/l`: (rainy) increasing character increases the condition
+  * `q`: Heavily clouded with rain
+  * `w`: (Heavily clouded with rain and winter precipitation)
+  * `m`: (Heavily clouded with some light rain)
+  * `u`: (Cloudy with light snow)
+  * `i/v`: (Heavily clouded with light snowfall) increasing character increases the condition
+  * `t`: (Heavy snowfall)
+  * `g`: (Clear with (possibly) some heavy lightning)
+  * `s`: (Cloudy with (possibly) some heavy (thunderstorms) showers)
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]:
```suggestion
  * `a`: sunny/clear
  * `b`: Mix of clear and medium or low clouds
  * `j`: Mix of clear and high clouds
  * `o/r`: Partly cloudy (increasing character increases the condition)
  * `c`: Heavily clouded
  * `p`: Cloudy
  * `d`: Alternating cloudy with local fog
  * `n`: Clear and local mist or fog
  * `f`: Alternatingly cloudy with some light rain
  * `h/k/l`: rainy (increasing character increases the condition)
  * `q`: Heavily clouded with rain
  * `w`: Heavily clouded with rain and winter precipitation
  * `m`: Heavily clouded with some light rain
  * `u`: Cloudy with light snow
  * `i/v`: Heavily clouded with light snowfall (increasing character increases the condition)
  * `t`: (Heavy snowfall)
  * `g`: (Clear with (possibly) some heavy lightning)
  * `s`: (Cloudy with (possibly) some heavy (thunderstorms) showers)
```

**Review by frenck — APPROVED** [review]
Thanks, @dennis-bell 👍

**Pull request merged**